web/src/pages/CreateRoom/CreateRoom.tsx

```diff
@@ -0,0 +1,73 @@
+import { Component, createSignal, Show } from "solid-js";
+import Button from "@/components/Button";
+
+import styles from "./CreateRoom.module.scss";
+import useCreateRoom from "./hooks/useCreateRoom";
+
+const defaultNumberOfOptions = 5;
+
+const CreateRoom: Component = () => {
+	const { fields, updateField, submit } = useCreateRoom();
+
+	const handleSubmit = (event: Event): void => {
+		event.preventDefault();
+		submit();
+	};
+
+	return (
+		<main class={styles.createRoom}>
+			<form class={styles.createRoomForm} onSubmit={handleSubmit}>
+				<label for="moderatorName">
+					Your name
+					<span class={styles.required}>*</span>
+				</label>
+
+				<input name="moderatorName" type="text" onInput={updateField} />
+
+				<div class={styles.seperator}>
+					<hr />
+				</div>
+
+				<label for="voterOptions">
+					What options would you like voters have to choose from?
+					<span class={styles.required}>*</span>
+				</label>
+
+				<select name="voterOptions" onInput={updateField}>
+					<option value="select">Select options</option>
```

Comment:
Things got a little funky for me when I chose the select value.  We probably want to go back and disable the slider and the done button if no option is selected.

Reply:
Def makes sense. I'll update that